app/scripts/controllers/finaltest/finaltest-controller.js

```diff
@@ -176,7 +176,7 @@
         showBackdrop: true,
         showDelay: 0
       });
-      ScreenshotService.getImage(view).then(function(imageBase64){
+      ScreenshotService.getImageBase64(view).then(function(imageBase64){
```

Comment:
```suggestion
      ScreenshotService.getBase64Image(view).then(function(imageBase64){
```
creo q está al revés el nombre. Habría q cambiarlo en el servicio.